tool/travis.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Fast fail the script on failures.
+set -e
+
+# Verify that the libraries are error free.
+dartanalyzer --fatal-warnings \
```

Comment:
You can likely just do `dartanalyzer --fatal-warnings .` here now.

Reply:
Yes, cool!